test/interface/named-node-map.js

```diff
@@ -2,10 +2,10 @@ const assert = require('../assert.js').for('NamedNodeMap');
 
 const {parseHTML} = global[Symbol.for('linkedom')];
 
-const {document} = parseHTML('<html><div tmp>abc</div></html>');
+const {document, NamedNodeMap} = parseHTML('<html><div tmp>abc</div></html>');
 
 let node = document.documentElement.firstElementChild;
-
+assert(typeof NamedNodeMap !== 'undefined', true, 'NamedNodeMap undefined in global export');
```

Comment:
My intention was to add a unit test related to the change here. I couldn't find similar checks for other exported objects but felt bad not adding something...